tools/pipelines/templates/build-npm-package.yml

```diff
@@ -425,7 +425,7 @@ extends:
                           targetFolder: $(Build.ArtifactStagingDirectory)/codeCoverageAnalysis
                       - task: Bash@3
                         displayName: Report Code Coverage Comparison
-                        condition: and(succeededOrFailed(), eq('${{ parameters.reportCodeCoverageComparison }}', true), eq(variables['ReportDirExists'], 'true'))
+                        condition: and(succeededOrFailed(), eq(parameters.reportCodeCoverageComparison, true), eq(variables['ReportDirExists'], 'true'))
```

Comment:
What was the problem? I don't think this is going to work, all access to `paramaters.*` needs to be inside template expressions `${{}}` AFAIK.

Reply:
The single quotes around `${{}}` were probably unnecessary though.